app.py

```diff
@@ -336,13 +338,18 @@ def generate_graph(street, direction, graph_name, day_type='Weekday', period='AM
               href = '/css/style.css'),
           
 html.Div(children=[html.H1(children=TITLE, id='title'),
+                   html.Button('info', id = 'info', className = 'info')
                   ], className='row twelve columns'),
     dcc.Tabs(tabs=[{'label': 'East-West Streets', 'value': 'ew'},
                    {'label': 'North-South Streets', 'value': 'ns'}],
              value='ew',
              id='tabs',
              style={'font-weight':'bold'})
     , html.Div(id=MAIN_DIV, className='row'),
+    html.Div(children = [
```

Comment:
This is looking good. If there's a way to give it rounded corners that would be nice, but not critical